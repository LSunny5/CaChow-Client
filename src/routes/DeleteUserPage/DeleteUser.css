/*delete page*/
.deleteContent {
    min-height: 92vh;
    padding: 2rem;
}

/*delete title*/
.deleteHeading {
    font-family: 'Cinzel', serif;
    font-size: 2rem;
    font-weight: bolder;
    margin: 1rem;
}

/*question to delete*/
.deleteMessage{
    font-family: 'Delius', cursive;
    width: 70%;
    font-size: 1.5rem;
    margin: 3rem;
}

/*container for delete buttons*/
.deleteButtonBox {
    display: flex;
    flex-direction: column;
    justify-content: space-evenly;
}

/*delete page buttons*/
.deleteButton {
    font-family: 'Delius', cursive;
    position: relative;
	padding: 10px 40px;
    margin: 1rem auto;
	border-radius: 8px;
	font-size: 25px;
    color: black;
    transition: all 0.1s;
    -webkit-transition: all 0.1s;
    width: 60%;
    border-top: none;
    border-left: none;
    cursor: pointer;
}

/*hover for delete button*/
.deleteButton:hover {
    filter: brightness(50%);
    color: white;
}

/*red 3d button look*/
.redshade {
    background-color: #E74C3C;
	border-bottom: 5px solid #BD3E31;
	text-shadow: 0px -2px #BD3E31;
}

/*yellow 3d button look*/
.yellowshade {
    background-color: #F2CF66;
	border-bottom: 5px solid #D1B358;
	text-shadow: 0px -2px #D1B358;
}

/********************************************************************/
/*For Ipads and larger screens */
@media only screen and (min-width: 600px) {
    /*delete title*/
    .deleteHeading {
        font-size: 3rem;
        margin: 3rem auto;
    }

    /*question to delete*/
    .deleteMessage{
        margin: 5rem auto;
    }

    /*container for delete buttons*/
    .deleteButtonBox {
        flex-direction: row;
    }

    /*delete page buttons*/
    .deleteButton {
        width: 30%;
    }
}

/********************************************************************/
/*For Laptops and larger screens */
@media only screen and (min-width: 1024px) {
    /*page for delete*/
    .deleteContent {
        min-height: 91vh;
    }
}