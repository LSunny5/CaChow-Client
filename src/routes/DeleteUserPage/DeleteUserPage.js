import React, { Component } from 'react';
import { NavLink } from 'react-router-dom';
import './DeleteUser.css';

export default class DeleteUserPage extends Component {
    render() {
        return (
            <section className='deleteContent'>
                <h2 className="deleteHeading">Deleting User...</h2>
                <div className="restaurantBox">
                    <p className="deleteMessage">
                        Are you sure you want to delete this account?
                    </p>
                    <div className = "buttonBox">


                        <NavLink to={'/login'} onClick={this.deleteRestaurant}>Delete</NavLink>
                        <NavLink to={'/ownerView'}>Cancel</NavLink>
                   
                   
                    </div>
                </div>
            </section >
        )
    };
}