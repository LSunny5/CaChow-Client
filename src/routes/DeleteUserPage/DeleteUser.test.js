import React from 'react';
import ReactDOM from 'react-dom';
import DeleteUserPage from './DeleteUserPage';
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'
import { BrowserRouter } from 'react-router-dom';

describe(`DeleteUserPage route`, () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        const props = {
            history: {
                push: () => { }
            }
        };
        ReactDOM.render(
            <BrowserRouter>
                <DeleteUserPage {...props} />
            </BrowserRouter>, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the DeleteUserPage form', () => {
        const wrapper = shallow(<DeleteUserPage />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
});