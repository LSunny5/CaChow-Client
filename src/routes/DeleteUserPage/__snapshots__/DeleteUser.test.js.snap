// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeleteUserPage route renders the DeleteUserPage form 1`] = `
<section
  className="deleteContent"
>
  <h2
    className="deleteHeading"
  >
    Deleting User...
  </h2>
  <div
    className="deleteBox"
  >
    <p
      className="deleteMessage"
    >
      Are you sure you want to 
      <span
        className="redandbold"
      >
        delete
      </span>
       this account?
    </p>
    <div
      className="deleteButtonBox"
    >
      <button
        className="deleteButton redshade redandbold"
        onClick={[Function]}
      >
        Delete
      </button>
      <button
        className="deleteButton yellowshade"
        onClick={[Function]}
      >
        Cancel
      </button>
    </div>
  </div>
</section>
`;
